src/test/java/tests/UITests/Test_EASEYIn_1088_Keep_state_of_facilities_among_tabs_in_Monitoring_Plans.java

```diff
@@ -0,0 +1,59 @@
+package tests.UITests;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.MonitoringPlansPage;
+import tests.utils.UITestBase;
+
+public class Test_EASEYIn_1088_Keep_state_of_facilities_among_tabs_in_Monitoring_Plans extends UITestBase {
+
+    @Test
+    public void tests () {
+
+        //Navigate to EASEY In
+        //https://easey-dev.app.cloud.gov/monitoring-plans
+
+        goTo("https://easey-dev.app.cloud.gov/monitoring-plans");
+
+        MonitoringPlansPage monitoringPlansPage = new MonitoringPlansPage(driver);
+
+        verifyEquals(monitoringPlansPage.title, "Monitoring Plans");
+
+        //Choose one facility from the table by clicking on the Open tab
+        // (Barry AL, Oris Code 3 was mainly used for this test) and select the open tab.
+        waitFor(driver -> monitoringPlansPage.tableResults.size() > 3);
+        String facility = monitoringPlansPage.tableResults.get(0).getText().split("\n")[1];
+        click(monitoringPlansPage.openFacilityTab.get(0));
+
+        //A new tab with the name of the selected facility should appear along with its Monitoring Plans data
+        verifyTrue(monitoringPlansPage.tabs.size() == 2);
+        click(monitoringPlansPage.tabs.get(1));
+        waitFor(monitoringPlansPage.facilityTitle);
+        verifyEquals(monitoringPlansPage.facilityTitle, facility);
+
+        // Selecting Monitoring Systems under Sections Dropdown
+        waitFor(monitoringPlansPage.monitoringMethods);
+        click(monitoringPlansPage.monitoringSystems);
+        Assert.assertEquals(true, monitoringPlansPage.monitoringSystems.isSelected());
```

Comment:
we have the assertTrue method in the framework that can be used. assertTrue(monitoringPlans.monitoringSystems.isSelected());